micro-rdk-installer/src/nvs/request.rs

```diff
@@ -22,7 +22,16 @@ pub async fn download_micro_rdk_release(
                 RELEASES_BASE_URL, "latest/download", BINARY_NAME
             )
         },
-        |version| format!("{}/download/{}/{}", RELEASES_BASE_URL, version, BINARY_NAME),
+        |version| {
```

Comment:
⬆️ - If you defaulted in `latest` above could you avoid the need to re-iterate the format block for the latest case?

Reply:
I can do this, was avoiding it for the clone, but as this is a cli, it doesn't really matter.
```rs
let release_url = if version.is_none() || version.clone().unwrap() == "latest" { ... } else { ... };
```

Better?

Reply:
Yes, I think so.